agents/zvm/fence_zvmip.py

```diff
@@ -163,21 +163,35 @@ def main():
 
 	docs = {}
 	docs["shortdesc"] = "Fence agent for use with z/VM Virtual Machines"
-	docs["longdesc"] = """The fence_zvm agent is intended to be used with with z/VM SMAPI service via TCP/IP
+	docs["longdesc"] = """The fence_zvmip agent is intended to be used with the z/VM SMAPI service via TCP/IP
 
-To  use this agent the z/VM SMAPI service needs to be configured to allow the virtual machine running this agent to connect to it and issue
-the image_recycle operation.  This involves updating the VSMWORK1 AUTHLIST VMSYS:VSMWORK1. file. The entry should look something similar to
-this:
+The z/VM SMAPI service must be configured to allow the virtual machine running the agent to connect to the service, access the system's
+directory manager, and issue the image_recycle operation. This involves updating the VSMWORK1 NAMELIST and VSMWORK1 AUTHLIST VMSYS:VSMWORK1
+files.
+
+The NAMELIST entry assigns all the required functions to one nick and should look similar to this:
+
+:nick.ZVM_FENCE
+:list.
+IMAGE_ACTIVATE
+IMAGE_DEACTIVATE
+IMAGE_STATUS_QUERY
+CHECK_AUTHENTICATION
+IMAGE_NAME_QUERY_DM
+
+
+The AUTHLIST entry authorizes the user to perform all the functions associated with the nick, and should look similar to this:
 
 Column 1                   Column 66                Column 131
 
-   |                          |                        |
-   V                          V                        V
+|                             |                        |
+V                             V                        V
+
+XXXXXXXX                      ALL                      ZVM_FENCE
```

Comment:
You should just indent this instead of undoing the arrow indentation above (which makes it look even more inconsistent).

Reply:
Agreed.